bqn-mode.el

```diff
@@ -266,6 +266,7 @@ BQN buffers (or recreate them)."
   (when bqn-glyph-map-modifier
     (set-keymap-parent bqn-mode-map
                        (make-composed-keymap prog-mode-map bqn--glyph-map)))
+  (keymap-set bqn-mode-map "C-c C-z" #'bqn-comint-bring)
```

Comment:
I split the keymap modifications because I think it is a little bit controversial to bind keys "by default". 

https://github.com/positron-solutions/user-keys?tab=readme-ov-file#please-package-authors

Reply:
I thought a little bit about this, but since keybindings of the form `C-c C-«letter»` are [reserved for major modes](https://www.gnu.org/software/emacs/manual/html_node/elisp/Key-Binding-Conventions.html) I figured it was fine. I think that it helps with discoverability, personally, and unbinding a keybinding from a keymap is not all that difficult. Though I don't feel too strongly either way and am fine with dropping this part if you'd prefer